### ISSUE TITLE
[BUG] Build failed with an exception.

### ISSUE THREAD

**Issue by aniketsongara** [tneotia/html-editor-enhanced#66]
I have updated the html_editor_enhanced: ^2.1.0 to html_editor_enhanced: ^2.2.0+1-dev.1 below build failed exception is coming 

```
Launching lib\main.dart on SM M105F in debug mode...
Running Gradle task 'assembleDebug'...
/H:/android%20studio/sdk/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_colorpicker-0.5.0/lib/src/hsv_picker.dart:730:43: Error: The parameter 'event' of the method 'AlwaysWinPanGestureRecognizer.addAllowedPointer' has type 'PointerDownEvent', which does not match the corresponding type, 'PointerEvent', in the overridden method, 'DragGestureRecognizer.addAllowedPointer'.
 - 'PointerDownEvent' is from 'package:flutter/src/gestures/events.dart' ('/H:/android%20studio/sdk/flutter/packages/flutter/lib/src/gestures/events.dart').
 - 'PointerEvent' is from 'package:flutter/src/gestures/events.dart' ('/H:/android%20studio/sdk/flutter/packages/flutter/lib/src/gestures/events.dart').
Change to a supertype of 'PointerEvent', or, for a covariant parameter, a subtype.
  void addAllowedPointer(PointerDownEvent event) {
                                          ^
/H:/android%20studio/sdk/flutter/packages/flutter/lib/src/gestures/monodrag.dart:256:8: Context: This is the overridden method ('addAllowedPointer').
  void addAllowedPointer(PointerEvent event) {
       ^


FAILURE: Build failed with an exception.

* Where:
Script 'H:\android studio\sdk\flutter\packages\flutter_tools\gradle\flutter.gradle' line: 1035

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'H:\android studio\sdk\flutter\bin\flutter.bat'' finished with non-zero exit value 1

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 47s
Exception: Gradle task assembleDebug failed with exit code 1

```

here my pubspec.yaml 

```
environment:
  sdk: ">=2.7.0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter
  cupertino_icons: ^1.0.3
  permission_handler: ^8.1.0
  flutter_html: ^2.1.0
  sqflite: ^2.0.0+3
  fluttertoast: ^8.0.7
  shimmer: ^2.0.0
  path_provider:
  device_apps: ^2.1.1
  flutter_spinkit: ^5.0.0
  device_info: ^2.0.2
  draggable_scrollbar: ^0.1.0
  intl: ^0.17.0
  http: ^0.13.3
  file_picker: ^3.0.2+2
  firebase_storage: ^8.1.3
  firebase_core: ^1.3.0
  html_editor_enhanced: ^2.1.1
  url_launcher: ^6.0.6
  firebase_dynamic_links: ^2.0.6
  dio: ^4.0.0
  chewie: ^1.2.2
  video_player: ^2.1.6
  record: ^2.1.1
  just_audio: ^0.7.5


dev_dependencies:
  flutter_test:
    sdk: flutter

flutter:
  uses-material-design: true
  assets:
    - images/image_default.png
    - images/internet_not_connected.png

```

**Comment by aniketsongara** [comment]
When I removed the html_editor_enhanced package from my pubspec.yaml then the app is building without any issue.

**Comment by tneotia** [comment]
Downgrade to 2.2.0:

![Screenshot_20210616-054927_Kiwi Browser.png](https://user-images.githubusercontent.com/50850142/122197903-a844cf80-ce66-11eb-815f-a8d88d9328a4.png)

Flutter stable will not work with 2.2.0+1 at the moment.

**Comment by aniketsongara** [comment]
I have downgraded it to 2.2.0 but still having the same exception.

**Comment by tneotia** [comment]
That shouldn't be happening - did you run pub get and flutter clean? 2.2.0 uses the 0.4.0 version of color picker so the error should not happen again.

**Comment by aniketsongara** [comment]
Thanks issue has been resolved when I have set 0.4.0 version of the color picker with editor 2.2.0 version. 
Why this error is coming in the latest version of the package and when it will be consider to be fixed ?

**Comment by tneotia** [comment]
There is a breaking change in Flutter relating to gestures, the new color picker version fixes that but is not compatible with stable. Anything beta+ will work with the latest version. 

I was trying to make the 2.2.0+1-dev.1 show up as a beta release but Pub made it a regular release for some reason. -dev should indicate beta use only.
